js/app/date-range-updater.js

```diff
@@ -1,38 +1,78 @@
-$(document).ready(function() {
-    $("#start-month").change(function() {
+$(document).ready(function () {
```

Comment:
Currently when you return to this page to edit what you saved previously, the End Year contains all options, so you can then select a year that precedes the Start Year. Should we implement the logic that filters out illogical End Years to also run on page load if the user is returning to it?

Reply:
Ha, that is a very good catch, looks like a bug we have in the Range selection too.